### PR TITLE
Add ci testing linux

### LINKED ISSUE #123: add CI testing for linux
breaking #110 into two parts

### PR DESCRIPTION
**Pull request by carlosmmatos**
Closes #123 

Add CI testing for the linux scripts using the molecule framework.....

### CODE REVIEW

File: .github/workflows/ci-linux.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+name: "Integration Tests: Linux"
+on:
+  schedule:
+    - cron: '0 6 * * *'
+
+  push:
```
Comment (ffalor) [inline .github/workflows/ci-linux.yml:6]: I might be wrong, but these paths (for pr too) should be updated to include the bash dir not just molecule

File: .github/workflows/ci-linux.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{github.event.pull_request.head.sha}}
+        if: github.event_name == 'pull_request_target'
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
```
Comment (ffalor) [inline .github/workflows/ci-linux.yml:57]: we can actually have setup-python cache our pip dependencies

add `cache: 'pip'`

**Review by ffalor — CHANGES_REQUESTED** [review]

**Review by ffalor — APPROVED** [review]

File: .github/workflows/ci-linux.yml (hunk 1 above)
Comment (carlosmmatos) [reply .github/workflows/ci-linux.yml:6]: you're right

File: .github/workflows/ci-linux.yml (hunk 2 above)
Comment (carlosmmatos) [reply .github/workflows/ci-linux.yml:57]: 👀

**Comment by ffalor** [comment]
lgtm

**Pull request merged**